src/SFA.DAS.Commitments.Api.Types/Commitment/TransferSenderInfo.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace SFA.DAS.Commitments.Api.Types.Commitment
+{
+    public class TransferSenderInfo
+    {
+        public long? TransferSenderId { get; set; }
```

Comment:
Prefixing "TransferSender" is superfluous, so we could have Id and Name. The last 3, i.e. TransferApprovalStatus on, are valid IMO, to distinguish between the types of approval.

Reply:
Done for id and name